avocado/plugins/run.py

```diff
@@ -166,9 +166,10 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_FAIL)
         job_instance = job.Job(args)
         job_run = job_instance.run()
-        result_dispatcher = ResultDispatcher()
-        if result_dispatcher.extensions:
-            result_dispatcher.map_method('render',
-                                         job_instance.result,
-                                         job_instance)
+        if job_instance.result.tests_total > 0:
+            result_dispatcher = ResultDispatcher()
+            if result_dispatcher.extensions:
+                result_dispatcher.map_method('render',
+                                             job_instance.result,
+                                             job_instance)
```

Comment:
Yep, drop the commit and fix the individual plugins (if they are not prepared, I never tired)